C-Sharp/other/Unique_Number_3.cs

```diff
@@ -0,0 +1,48 @@
+//Unique_Number_III
+//Given an array that contains all elements occurring 3 times, but one occurs only once. Find that unique element.
+
+using System.IO;
+using System;
+
+class Neoalgo {
```

Comment:
@rutujadhanawade Rename the class name as `Unique_Number_III`.

Reply:
Done